packages/react-components/src/components/styles/ParticipantItem.styles.ts

```diff
@@ -3,6 +3,8 @@
 
 import { mergeStyles } from '@fluentui/react';
 
+const MINIMUM_TOUCH_TARGET_HEIGHT_REM = 3;
```

Comment:
Can we extract this constant (and the one in ChatMessageComponent.styles.ts) to a common file to be used from all styles files in `react-components`?